src/less/chat.less

```diff
@@ -23,6 +24,118 @@
   }
 }
 
+.chat-card-custom {
+  .chat-card-custom-item{
+    img{
+      margin:5px;float:left;
```

Comment:
`css` properties should be on separate lines.

Reply:
done